homeassistant/components/invoxia/device_tracker.py

```diff
@@ -0,0 +1,136 @@
+"""Platform for invoxia.device_tracker integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from gps_tracker import AsyncClient, Tracker
+from gps_tracker.client.datatypes import Tracker01
+
+from homeassistant.components import device_tracker
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ENTITIES
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    ATTRIBUTION,
+    CLIENT,
+    COORDINATORS,
+    DOMAIN,
+    LOGGER,
+    MDI_ICONS,
+    TRACKERS,
+)
+from .coordinator import GpsTrackerCoordinator
+from .helpers import GpsTrackerData
+
+PARALLEL_UPDATES = 1
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the device_tracker platform."""
+    entry_data = hass.data[DOMAIN][entry.entry_id]
+    client: AsyncClient = entry_data[CLIENT]
+    trackers: list[Tracker] = entry_data[TRACKERS]
+    coordinators: list[GpsTrackerCoordinator] = entry_data[COORDINATORS]
+
+    entities = [
+        GpsTrackerEntity(coordinator, client, tracker)
+        for tracker, coordinator in zip(trackers, coordinators)
+    ]
+
+    entry_data[CONF_ENTITIES].extend(entities)
+    async_add_entities(entities, update_before_add=True)
+
+
+@dataclass(frozen=True)
+class GpsTrackerEntityDescription(EntityDescription):
+    """Describes a Invoxia GPS tracker entity."""
+
+
+class GpsTrackerEntity(
+    CoordinatorEntity[GpsTrackerCoordinator], device_tracker.TrackerEntity
```

Comment:
Please adopt the entity naming specified in https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations